timesketch/views/auth.py

```diff
@@ -36,18 +40,39 @@
 def login():
     """Handler for the login page view.
 
-    There are two ways of authentication.
-    1) If Single Sign On (SSO) is enabled in configuration and the environment
+    There are three ways of authentication.
+    1) Google Cloud Identity-Aware Proxy.
+    2) If Single Sign On (SSO) is enabled in configuration and the environment
        variable is present, e.g. REMOTE_USER then the system will get or create
        the user object and setup a session for the user.
-    2) Local authentication is used if SSO login is not enabled. This will
+    3) Local authentication is used if SSO login is not enabled. This will
        authenticate the user against the local user database.
 
     Returns:
         Redirect if authentication is successful or template with context
         otherwise.
     """
-    form = UsernamePasswordForm()
+
+    # Google Identity-Aware Proxy authentication (using JSON Web Tokens)
+    if current_app.config.get(u'GOOGLE_IAP_ENABLED', False):
```

Comment:
import unicode_literals and forget about the u's

Reply:
Ack. Same comment as above, will fix in an upcoming PR.